core/src/main/resources/default-configs/default-jgroups-azure.xml

```diff
@@ -60,5 +61,5 @@
    <MFC max_credits="4m"
         min_threshold="0.40"
    />
-   <FRAG4/>
+   <FRAG4 frag_size="${jgroups.frag_size:60000}"/>
```

Comment:
If we are going to use the default how about `<FRAG4 frag_size="${jgroups.frag_size}"/>`?
If the property wasn't set, then the default will be used

Reply:
this way you know exactly what is the value instead of having to go through the JGroups code.

Reply:
and a warning `ISPN000901: Property jgroups.frag_size could not be replaced as intended!`

Reply:
This actually brings up a good point that it would be nice if there was a way to configure use the jgroups default unless overridden instead of an explicit value that can be overridden.  It is very likely that while these values are the same as the jgroups default that these values will drift apart over time.

Reply:
@belaban Is there any way to configure that today?

Reply:
We are parsing that. Currently, if a property cannot be replaced, we issue a warning (`ISPN-901 Property %s could not be replaced as intended!`), which keeps the string as is. We could instead take that as "use the default"

Reply:
To avoid changing the above behaviour, we could use a special symbol for the `v` in `${p:v}` which means "if the property cannot be replaced, use the default". Something like `<FRAG4 frag_size="${jgroups.frag_size:$!}"/>"

Reply:
Sounds good to me. I wasn't trying to have this block the PR, so thanks for integrating.